juri/weather_widget/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather widget</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="api">
+        <div class="container">🌞 This demo needs an OpenWeather API key to work. <a target="_blank" href="https://home.openweathermap.org/users/sign_up">Get yours here for free!</a>
+        </div>
+      </div>
+      <section class="top-banner">
+        <div class="container">
+          <h1 class="heading">Simple Weather App</h1>
+          <form>
+            <input type="text" placeholder="Search for a city" autofocus>
```

Comment:
Consider using https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search
```suggestion
          <input type="search" placeholder="Search for a city" autofocus />
```